ocs/agents/influxdb_publisher/drivers.py

```diff
@@ -88,6 +92,9 @@ def __init__(self, host, database, incoming_data, port=8086, protocol='line', gz
                 LOG.error("Connection error, attempting to reconnect to DB.")
                 self.client = InfluxDBClient(host=self.host, port=self.port, gzip=gzip)
                 time.sleep(1)
+            if operate_callback and not operate_callback():
+                break
```

Comment:
This is neat, but I don't quite follow how this allows a ctrl+c interruption. Does the ctrl+c result in a `stop` call, which makes `self.aggregate == False` so `operate_callback()` is `False`, then we hit the `break`?

Reply:
That's alright, just wanted to make sure I understood what was going on. Thanks for confirming!